.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't know what this file is but it doesn't belong here.

Reply:
I'm extremely sorry that this got committed. It's from my local editor. I'll remove it right away.

P.s. Would you be okay with a separate PR for a .gitignore that excludes the usual stuff like this?

Reply:
A .gitignore for the specifications repository would be fine.
It could also ignore the META-INF directory that sometimes
is accidentally included with the javadocs.